Sources/Tools/plutil/CMakeLists.txt

```diff
@@ -2,6 +2,30 @@ add_executable(plutil
   main.swift)
 target_link_libraries(plutil PRIVATE
   Foundation)
+
+# On ELF platforms, remove the absolute rpath to the host toolchain's stdlib, then add it
+# back temporarily as a BUILD_RPATH just for the tests.
+if(NOT CMAKE_SYSTEM_NAME MATCHES "Darwin|Windows")
+  target_link_options(plutil PRIVATE "SHELL:-no-toolchain-stdlib-rpath")
```

Comment:
There's a plutil test that fails without this rpath to the host toolchain's stdlib, so add it back in as a `BUILD_RPATH` below, ie an rpath that's removed by CMake when the binary is installed.